.github/workflows/early-access.yml

```diff
@@ -47,15 +40,12 @@ jobs:
         if: ${{ runner.os == 'Windows' }}
         uses: ilammy/msvc-dev-cmd@v1
 
-      - name: 'Set up Graal'
-        uses: DeLaGuardo/setup-graalvm@5.0
+      - uses: graalvm/setup-graalvm@v1
         with:
-          graalvm: '21.3.0'
-          java: 'java17'
-
-      - name: 'Install native-image component'
-        run: |
-          ${{ matrix.gu-binary }} install native-image
+          version: ${{ matrix.version }}
+          java-version: '17'
+          components: 'native-image'
+          github-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Do you know why the token is needed here?

Reply:
Written in [docs](https://github.com/marketplace/actions/github-action-for-graalvm, ), to avoid rate limiting